app/views/layout.erb

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Space</title>
+    <link rel='stylesheet' type='text/css' href='/css/style.css'>
+    <link href="/vendors/css/bootstrap.min.css" rel="stylesheet">
+    <link href='https://fonts.googleapis.com/css?family=Lato' rel='stylesheet' type='text/css'>
+    <link href='https://fonts.googleapis.com/css?family=Nunito' rel='stylesheet' type='text/css'>
+
+    <script type='text/javascript' src='/vendors/js/jquery.js'></script>
+    <script src="/vendors/js/bootstrap.min.js"></script>
+    <script src="/vendors/js/jquery-ui.js"></script>
+  </head>
+
+  <body>
+    <header class="jumbotron">
+      <div class="container text-center">
+        <h1>Space</h1>
+        <h2>There is beauty in space</h2>
+      </div>
+    </header>
+    <main>
+
+      <%= partial :sign_up_errors %>
+
+      <%= partial :sign_out %>
+
+      <%= yield %>
+
+
+    </main>
+
+    <script>
+
+    $( document ).ready(function() {
+
+      $('#date_picker').datepicker();
+
+      $(function() {
+
+        $( "#date_picker_from" ).datepicker({
+          dateFormat: "dd/mm/yy",
+          defaultDate: "+1w",
+          changeMonth: true,
+          minDate: 0,
+          onClose: function( selectedDate ) { $( "#date_picker_to" ).datepicker( "option", "minDate", selectedDate ); }
+        });
+
+        $( "#date_picker_to" ).datepicker({
+          dateFormat: "dd/mm/yy",
+          defaultDate: "+1w",
+          changeMonth: true,
+          onClose: function( selectedDate ) {  $( "#date_picker_from" ).datepicker( "option", "maxDate", selectedDate ); }
+        });
+
+      });
+    });
+
+
+
+
```

Comment:
Neat indentation but a lot of random whitespace. Fred George says: "a blank line is a comment that signals intent." Don't overuse them!